VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.html

```diff
@@ -1,276 +1,124 @@
-<div class="container" *ngIf="participants.length > 0">
-  <h2 class="govuk-heading-s white">{{ 'participants-panel.participants' | translate }} ({{ participants.length }})</h2>
+<div class="container" *ngIf="totalParticipants > 0">
+  <div class="header-row">
+    <p class="govuk-body white">{{ 'participants-panel.participants' | translate }} ({{ totalParticipants }})</p>
+    <a
+      id="totalInWrCount"
+      class="govuk-body white vh-pointer"
+      tabindex="0"
+      (keydown.enter)="scrollToElement('panelListNotConnectedHeader')"
+      (click)="scrollToElement('panelListNotConnectedHeader')"
+    >
+      {{ 'participants-panel.in-waiting-room' | translate }} ({{ totalParticipantsInWaitingRoom }})
+    </a>
+  </div>
 
   <div class="button-row">
-    <div
+    <button
+      type="submit"
```

Comment:
do these buttons still work? You've changed them to a submit, but there's no form

Reply:
yup, I should probable make them normal button types though

Reply:
oh my bad I've been testing this with the keyboard only! great catch Oliver!